src/NServiceBus.SagaAudit.Tests/APIApprovals.cs

```diff
@@ -14,20 +11,7 @@ public class APIApprovals
     [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
You can remove this when using Particular.Approvals.

Reply:
Thanks, I missed one.